tests/testthat/test-rstudio_markers.R

```diff
@@ -10,11 +13,8 @@ test_that("it returns markers which match lints", {
     ),
     class = "lints"
   )
-  with_mock(
-    `rstudioapi::callFun` = function(...) list(...),
-    `rstudioapi::executeCommand` = function(...) NULL,
-    marker1 <- rstudio_source_markers(lint1)
-  )
+
+  marker1 <- rstudio_source_markers(lint1)
```

Comment:
Checking this led me to notice Commit d657e42ad95828dec06d9a147508a55b84727858 introduced a name clash in `print.lints`. Can anyone with RStudio on hand confirm `print.lints` no longer working since that commit?

Reply:
nvm. To my surprise, R allows such clashes, although very misleading:

```r
x <- function() cat("ok\n")
local({x <- 2; x()})
# > ok
```

Reply:
yes for calls R essentially does get(mode = 'function')

Reply:
Agree it's better to avoid this behavior, but also out of scope for this PR; saw the follow-up, thanks.